### ISSUE TITLE
enabling prototype classes break sass compilation

### ISSUE THREAD

**Issue by swi2012** [acoustep/ember-cli-foundation-6-sass#82]
if i set in my app.sass
`@include foundation-everything($prototype: true);`
then i get error on compilation
`The Broccoli Plugin: [BroccoliMergeTrees] failed with:
Error: You may not @extend an outer selector from within @media.
You may only @extend selectors within the same directive.
From "@extend h1" on line 15 of tmp/sass_compiler-input_base_path-E59EwuML.tmp/1/foundation-sites/prototype/_typescale.scss
`
ember-cli-foundation-6-sass@0.0.24

**Comment by GCheung55** [comment]
I'm wondering if this is a bug in Foundation. What version of Foundation do you have installed?

**Comment by swi2012** [comment]
I have foundation 6.4.1. Same version i have for my other project (just standart foundation cli project that build sass to css) and there is builded perfectly. As i understand in both sass compiled with node-sass.
The main difference i see is version of node-sass. Ember use lates 4.5.3, but foundation cli project use node-sass 3.13.1

**Comment by swi2012** [comment]
Indeed. When i set "node-sass": "^3.13.0" in package.json of broccoli-sass-source-maps and make npm install (to get 3.13 version of node-sass) it' compiled ok. So, what lib to be blame (post bug report to)? node-sass?

**Comment by GCheung55** [comment]
@swi2012 can you share your `app.scss`? I'm going to try and reproduce the issue.

**Comment by GCheung55** [comment]
I believe this is an issue in Foundation. See https://github.com/zurb/foundation-sites/issues/10417.

The issue should be fixed once foundation-sites@6.4.2 is released.

**Comment by GCheung55** [comment]
I believe this is an issue in Foundation. See https://github.com/zurb/foundation-sites/issues/10417.

The issue should be fixed once foundation-sites@6.4.2 is released, but we should keep this issue open until then.

**Comment by IamManchanda** [comment]
6.4.2 is released now... Please check thanks!
